web/modules/custom/dpl_patron_reg/src/Controller/DplPatronRegController.php

```diff
@@ -176,4 +179,40 @@ public function userRegistrationReactAppLoad(): array {
     return $render;
   }
 
+  /**
+   * Make sure that user is detected as registered after registration.
+   *
+   * @param \Symfony\Component\HttpFoundation\Request $request
+   *   Symfony request object.
+   *
+   * @return \Symfony\Component\HttpFoundation\RedirectResponse
+   *   A redirect to the authorization endpoint.
+   */
+  public function postRegister(Request $request): RedirectResponse {
+    $access_token = $this->unregisteredUserTokensProvider->getAccessToken();
+    $logger = $this->getLogger('dpl_patron_reg');
```

Comment:
It is possible to go through the oauth flow again. I did that in my first version of the code.
But as @Dresse pointed out we can actually used the first token issued by Adgangsplatformen, it is usable for requesting services after registration at FBS.
So my thought was that, since it is not necessary, we can avoid another roundtrip with possible errors in the redirects and extra load to Adgangsplatformen.